bin/kafka_broker_generator.py

```diff
@@ -108,7 +143,10 @@
     spec:
       containers:
         - name: receiver
-          image: quay.io/openshift-knative/eventing-hyperfoil-benchmark-vertx-receiver
+          image: ghcr.io/pierdipi/sacura/sacura-7befbbbc92911c6727467cfbf23af88f
```

Comment:
we talked about this.

Perhaps worth to capture gaps somewhere, and why we use the sacura

Reply:
+1, will make sure to capture this in the doc